pkg/dockerregistry/server/auth.go

```diff
@@ -415,17 +449,37 @@ func verifyPruneAccess(ctx context.Context, client client.SubjectAccessReviews)
 			Resource: "images",
 		},
 	}
-	response, err := client.SubjectAccessReviews().Create(&sar)
-	if err != nil {
-		context.GetLogger(ctx).Errorf("OpenShift client error: %s", err)
-		if kerrors.IsUnauthorized(err) || kerrors.IsForbidden(err) {
+	return withExponentialBackoff(func() error {
```

Comment:
this is concerning to me... I can see retry requests in cases where we expect conflicts, like create/update, but not here